int8 Eme_stop_state
int8 Steering_state
int8 DBS_state
int8 Remote_state
int8 iECU_state
float32 Vehicle_speed
float32 Vehicle_break
float32 Tire_angle
int8 Vehilce_gear
int8 DriveMode_state



